runtime/android/core/src/org/xwalk/core/XWalkApplication.java

```diff
@@ -9,6 +9,8 @@
 import android.content.res.Resources;
 
 /**
+ * This class is deprecated.
```

Comment:
Although this class has been released for a while, but it's only for shared mode, and shared mode was just released from crosswalk-14, so this class isn't really used. So we just hide this class from the java docs instead marking it as deprecated